docs/confGenerator.md

```diff
@@ -30,6 +30,9 @@ Flags:
 
 ```
 
+> Note: confgenerator is available also from `netobserv/flowlogs2metrics` quay image. To use execute:  
```

Comment:
Didn't we change `flowlogs2metrics` to `flowlogs-pipeline`?

Reply:
fixed